config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -207,7 +207,7 @@ periodics:
           - name: GINKGO_FOCUS
             value: "API Version Upgrade"
           - name: KUBERNETES_VERSION
-            value: "v1.27.16"
+            value: "v1.28.15"
```

Comment:
Should we update to latest `v1.30.x` instead ?

Reply:
`v1.30` is the `n-2` version of the latest K8s release of `v1.32.3` supported by AKS.

Reply:
> Should we update to latest v1.30.x instead ?

Yes, I think we should stick with supported versions of Kubernetes, we just haven't been keeping this up-to-date.